src/__tests__/CompareResults/ResultsView.test.tsx

```diff
@@ -164,12 +170,10 @@ describe('Results View', () => {
         results: [testData[0]],
       });
 
-    const mockGetRandomValues = jest.fn().mockReturnValue(new Uint32Array(1));
-    Object.defineProperty(window, 'crypto', {
-      value: { getRandomValues: mockGetRandomValues },
-    });
+    window.alert = jest.fn();
```

Comment:
nit: it's cleaner to do:
```js
jest.spyOn(window, "alert").mockImplementation(() => {});
```
so that the original version is automatically restored at the end of the test.